include/gsl/span

```diff
@@ -731,7 +775,8 @@ as_bytes(span<ElementType, Extent> s) noexcept
     // clang-format off
     GSL_SUPPRESS(type.1) // NO-FORMAT: attribute
     // clang-format on
-    return {reinterpret_cast<const byte*>(s.data()), s.size_bytes()};
+    return span<const byte, details::calculate_byte_size<ElementType, Extent>::value>(
```

Comment:
Style: Since you're paying 2 lines anyways, consider `using type =` to simplify the return statement. Ditto `as_writable_bytes`.

Reply:
Good call, swapping that in. Thanks.